packages/react-client/src/ReactFlightClient.js

```diff
@@ -76,53 +76,63 @@ const RESOLVED_MODULE = 'resolved_module';
 const INITIALIZED = 'fulfilled';
 const ERRORED = 'rejected';
 
+// Dev-only
+type ReactDebugInfo = Array<{+name?: string}>;
```

Comment:
Why `Array`? Could there be a collection with a size > 1 for a chunk?

Reply:
Yea. For one chunk you can have multiple Server Components. Like a Server Component rendering another Server Component and so on. There will also be possible to have other steps inside like promises resolving. There will also be logs which has to be associated with the right component so order matters.

Basically all kinds of things that happened will be expressed in the order they happened.